### ISSUE TITLE
Unit tests for tourbook import/export function 

### ISSUE THREAD

**Issue by chrgernoe** [YacGroup/yacguide#390]
Can be started after #388 is merged.

Add tests for classes:
* JsonImporter
* JsonExporter
* CvsExporter
* TourbookExporterFactory

**Comment by paetz** [comment]
Correct me if I'm wrong but since we derive the What'sNew info from the issues that are tagged with the current milestone, we should not tag ktlo issues with a milestone. The user does not need (and does not want) to know anything about test, e.g. 
The user is interested in features and of course, a delivered feature counts as fully-tested for the user :)

**Comment by chrgernoe** [comment]
No, we don't derive it from there. We just take all PRs since the last release. Since I write the official What's New in German, I anyway go through all the issues and sort-out KTLO issues.
I tagged the issue with `v0.7.0`, so that we don't forget it.

**Comment by paetz** [comment]
Ok then I will tag all unit test I've done, too.
